examples/docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Great! Could you add a `README.md` in `examples/docker-compose` with the basic instructions on how to use it to run the example and perhaps check logs etc.